reference-architecture/azure-ansible/bastion.sh

```diff
@@ -325,6 +325,11 @@ cat > /home/${AUSERNAME}/setup-azure-node.yml <<EOF
   - name: make sure its set to unsceduable
     command: oadm manage-node {{inventory_hostname}} --schedulable=false
     delegate_to: master1
+- hosts: infranodes
+  tasks:
+  - name: make sure its set to unsceduable
+    command: oadm manage-node {{inventory_hostname}} --schedulable=false
```

Comment:
If the infranodes are not schedulable they won't be able to host the registry/router/etc.